app/scripts/modules/core/src/validation/ValidationError.tsx

```diff
@@ -1,11 +1,7 @@
 import * as React from 'react';
 
-export class ValidationError extends React.Component<{ message: string }> {
-  public render() {
-    return (
-      <div className="error-message">
-        <span className="fa fa-exclamation-circle" /> {this.props.message}
-      </div>
-    );
-  }
-}
+export const ValidationError = (props: { message: string | JSX.Element }) => (
+  <div className="error-message">
+    <span className="fa fa-exclamation-circle" /> {props.message}
```

Comment:
![image](https://user-images.githubusercontent.com/2153660/41257553-22f33288-6d82-11e8-8377-c7d49e9168aa.png)
Semantic proposes the above for the error message, mostly to give it a little room and explain what is wrong and what one can do. They are called messages. I like how they scale for different usecases. 
https://semantic-ui.com/collections/message.html